src/services/utilities.ts

```diff
@@ -92,7 +92,11 @@ namespace ts {
         if (node.kind === SyntaxKind.SourceFile) {
             return SemanticMeaning.Value;
         }
-        else if (node.parent.kind === SyntaxKind.ExportAssignment || node.parent.kind === SyntaxKind.ExternalModuleReference) {
+        else if (node.parent.kind === SyntaxKind.ExportAssignment
+            || node.parent.kind === SyntaxKind.ExternalModuleReference
+            || node.parent.kind === SyntaxKind.ImportSpecifier
+            || node.parent.kind === SyntaxKind.ImportClause
+            || isImportEqualsDeclaration(node.parent) && node === node.parent.name) {

```

Comment:
The rename was bailing because it was calling `getMeaningFromLocation` on the import specifier’s identifier, and it was making it all the way to the bottom of the function, returning `SemanticMeaning.Value`. Seems like identifier serving as an import alias should be `SemanticMeaning.All`.